flybywiresim/a32nx
# fix(displays): backlight bleed lighting cleanup and removed home cockpit mode

Fixes #7854
Fixes #7936 

## Summary of Changes

- Fix ISIS background being clear when A/C is powered off
- Fix some background elements still using #040405/ RGB(4,4,5) instead of 040404/RGB(4,4,4)
- Removed home cockpit mode as it is now has no effect (backlight bleed is no longer done through css or in code at all)
- Hotfix for self-test being non-functional (not a proper fix yet)
- New glass texture + comp (sans dust) from Big Mike

## Screenshots (if necessary)



![image](https://user-images.githubusercontent.com/15316958/230717455-50e53ada-e974-4a09-9e1f-b1a35a63802d.png)
![image](https://user-images.githubusercontent.com/15316958/230717489-4749080e-7fab-4cd9-929b-a5bf22facc43.png)
![image](https://user-images.githubusercontent.com/15316958/230717516-40e85059-fbcd-493b-9fda-c71bef80d2e3.png)
![image](https://user-images.githubusercontent.com/15316958/230717520-9d52684f-d121-46d8-b28c-5fc6b8e443cb.png)

## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


**NOTE: Self-Test is currently broken in Dev, this PR does not fix it**

- Load in several spawn states (C/D, CRZ, Final approach, etc.) and ensure that all the MFDs (PFD/ND), the flyPad EFB and ECAM display elements is working correctly.
- When the A/C is powered off, pop-out all displays with RALT+Click and ensure that the backgrounds are all BLACK and not transparent/reflective.
- Check that self-test is working (include 10s standby time from when displays are turned off. If more than 10 seconds, this will re-trigger self-test)
- Check that interior lighting (pedestal, floods, etc) still works as before.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
